grml2usb

```diff
@@ -273,28 +266,11 @@ class VerifyException(Exception):
     @Exception: message"""
 
 
-# The following two functions help to operate on strings as
-# array (list) of bytes (octets). In Python 3000, the bytes
-# datatype will need to be used. This is intended for using
-# with manipulation of files on the octet level, like shell
-# arrays, e.g. in MBR creation.
-
-
-def array2string(*a):
-    """Convert a list of integers [0;255] to a string."""
-    return struct.pack("%sB" % len(a), *a)
-
-
-def string2array(s):
-    """Convert a (bytes) string into a list of integers."""
-    return struct.unpack("%sB" % len(s), s)
-
-
 def cleanup():
     """Cleanup function to make sure there aren't any mounted devices left behind.
     """
 
-    def del_failed(fn, path, exc):
+    def del_failed(_fn, path, _exc):
         logging.warn("Deletion of %s failed in temporary folder" % path)
```

Comment:
```suggestion
        logging.warning("Deletion of %s failed in temporary folder", path)
```

Reply:
Uh nice find, thanks! Fixed in new version